assets/src/components/books/Book.test.jsx

```diff
@@ -180,6 +181,28 @@ describe('Book', () => {
       expect(leaveWaitlist).toHaveBeenCalledWith(book);
       expect(bookComponent).toHaveJoinWaitlistButton();
     });
+
+    it('has a waitlist indicator when book is on users waitlist', async () => {
+      const book = someBookThatIsInMyWaitlist();
+      const bookComponent = createComponent(book);
+
+      const indicator = bookComponent.find(WaitlistIndicator);
```

Comment:
I think we should avoid querying for a component. Let's use test id's instead. That way we don't test the implementation itself.

Reply:
adfd63f